### ISSUE TITLE
GetChampions throws Newtonsoft.Json.JsonSerializationException when selected language is de_DE

### ISSUE THREAD

**Issue by CommanderJo** [BenFradet/RiotSharp#355]
GetChampions() throws a Newtonsoft.Json.JsonSerializationException when called with the following parameters:

`_statiRiotApi.GetChampions(Region.euw, ChampionData.all, Language.de_DE)`

Does not occur when `en_US` is selected as a language or when an other value than `ChampionData.all` is used.

**Comment by BenFradet** [comment]
Mmh interesting, can you paste the full expcetion here?

**Comment by CommanderJo** [comment]
```
Newtonsoft.Json.JsonSerializationException was unhandled by user code
  HResult=-2146233088
  Message=Error setting value to 'Partype' on 'RiotSharp.StaticDataEndpoint.ChampionStatic'.
  Source=Newtonsoft.Json
  StackTrace:
       at Newtonsoft.Json.Serialization.DynamicValueProvider.SetValue(Object target, Object value)
       at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.SetPropertyValue(JsonProperty property, JsonConverter propertyConverter, JsonContainerContract containerContract, JsonProperty containerProperty, JsonReader reader, Object target)
       at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
       at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
       at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
       at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateDictionary(IDictionary dictionary, JsonReader reader, JsonDictionaryContract contract, JsonProperty containerProperty, String id)
       at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
       at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
       at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.SetPropertyValue(JsonProperty property, JsonConverter propertyConverter, JsonContainerContract containerContract, JsonProperty containerProperty, JsonReader reader, Object target)
       at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
       at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
       at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
       at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
       at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
       at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
       at Newtonsoft.Json.JsonConvert.DeserializeObject[T](String value, JsonSerializerSettings settings)
       at RiotSharp.StaticRiotApi.GetChampions(Region region, ChampionData championData, Language language)
       at Opstats.Web.Areas.Bravery.Controllers.HomeController.Index() in E:\TFS Opstats\Opstats\Main\OpStats.Web\Areas\Bravery\Controllers\HomeController.cs:line 46
       at lambda_method(Closure , ControllerBase , Object[] )
       at System.Web.Mvc.ActionMethodDispatcher.Execute(ControllerBase controller, Object[] parameters)
       at System.Web.Mvc.ReflectedActionDescriptor.Execute(ControllerContext controllerContext, IDictionary`2 parameters)
       at System.Web.Mvc.ControllerActionInvoker.InvokeActionMethod(ControllerContext controllerContext, ActionDescriptor actionDescriptor, IDictionary`2 parameters)
       at System.Web.Mvc.Async.AsyncControllerActionInvoker.<BeginInvokeSynchronousActionMethod>b__39(IAsyncResult asyncResult, ActionInvocation innerInvokeState)
       at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResult`2.CallEndDelegate(IAsyncResult asyncResult)
       at System.Web.Mvc.Async.AsyncResultWrapper.WrappedAsyncResultBase`1.End()
       at System.Web.Mvc.Async.AsyncControllerActionInvoker.EndInvokeActionMethod(IAsyncResult asyncResult)
       at System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<InvokeActionMethodFilterAsynchronouslyRecursive>b__3d()
       at System.Web.Mvc.Async.AsyncControllerActionInvoker.AsyncInvocationWithFilters.<>c__DisplayClass46.<InvokeActionMethodFilterAsynchronouslyRecursive>b__3f()
  InnerException: 
       HResult=-2147467261
       Message=Object reference not set to an instance of an object.
       Source=RiotSharp
       StackTrace:
            at SetPartype(Object , Object )
            at Newtonsoft.Json.Serialization.DynamicValueProvider.SetValue(Object target, Object value)
       InnerException: 

```

**Comment by BenFradet** [comment]
Woops, I didn't know that [ParTypes](https://github.com/BenFradet/RiotSharp/blob/develop/RiotSharp/StaticDataEndpoint/Champion/Enums/ParTypeStatic.cs) were translated.

As a result it doesn't work for any other language than english at the moment.

The solution would be to completely remove the ParType enum, good news is that it's in the roadmap and being discussed in #307.

**Comment by BenFradet** [comment]
If you want to help out, feel free to open a PR removing the ParType enum and turning it into a string!

**Comment by CommanderJo** [comment]
Yes, I can do that.

**Comment by BenFradet** [comment]
Great, looking forward to your PR :+1:

**Comment by BenFradet** [comment]
fixed in #356
